assignments/closure.js

```diff
@@ -1,19 +1,39 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function rapperString() {
+  return(`lil ${string} `);
+}
+let string = 'bunny';
+console.log(rapperString());
 
```

Comment:
I think I know what you were getting at with this but I don't know that its the most explicit or simple example. I'd need to have you explain it to me to ensure that you really understand the concept, which I expect you do. My point is that a different example might more clearly show your understanding of the idea. This was the solution example:
```
let myName = "Josh";

function sayHello() {
  let example = "jim";
  // accessing myName outside this function
  return `Hello ${myName}`;
}
console.log(sayHello());

console.log(example);
```